services/ui-src/src/utils/forms/forms.ts

```diff
@@ -311,8 +320,18 @@ export const updateRenderFields = (
   fields: (FormField | FormLayoutElement)[]
 ) => {
   const targetPopulations = report?.fieldData?.targetPopulations;
-  const filteredTargetPopulations =
-    removeNotApplicablePopulations(targetPopulations);
+
+  const notApplicablePopulation = {
+    id: "3Nc13O5GHA6Hc4KheO5FMSD2",
+    transitionBenchmarks_targetPopulationName: "Not applicable",
+    type: "targetPopulations",
+    isRequired: false,
+  };
+
+  const filteredTargetPopulations = new Array(
+    ...removeNotApplicablePopulations(targetPopulations),
+    notApplicablePopulation
```

Comment:
comment: these two names are unfortunate together (remove not applicable, then next line you add "notApplicable") but I understand what is happening

Reply:
ha yea good point. I just updated it to notApplicableOption.